### PR TITLE
Remove some unused imports and dependencies.

### PR DESCRIPTION
**Pull request by egrimley-arm**

### CODE REVIEW

**Comment by mathias-arm** [comment]
As mentioned in #358, I think it would be valuable for this PR to be rebase after #358 is base to see the impact. Otherwise good clean-ups.

**Review by mathias-arm — APPROVED** [review]

File: execution-engine/src/fs.rs [hunk 1]
```diff
@@ -21,13 +21,14 @@ use policy_utils::{
     CANONICAL_STDERR_FILE_PATH, CANONICAL_STDIN_FILE_PATH, CANONICAL_STDOUT_FILE_PATH,
 };
 #[cfg(not(feature = "icecap"))]
+use std::ffi::OsString;
```
Comment (dominic-mulligan-arm) [inline execution-engine/src/fs.rs:24]: Can this be simplified to a single import?

```rust
#[cfg(not(feature = "icecap"))]
use std::{ffs::OsString, os::unix::ffi::{OsStrExt, OsStringExt}};
```

**Review by dominic-mulligan-arm — COMMENTED** [review]
Looks good to me, barring a small change.

File: execution-engine/src/fs.rs (hunk 1 above)
Comment (egrimley-arm) [reply execution-engine/src/fs.rs]: Done, but I must check `icecap` on `localci` after the rebase.

**Review by dominic-mulligan-arm — APPROVED** [review]

File: execution-engine/src/fs.rs (hunk 1 above)
Comment (dominic-mulligan-arm) [reply execution-engine/src/fs.rs]: Great, thanks!

**Pull request merged**